index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Lawffice</title>
+</head>
+<body>
+    <header>
+        <h1>Law Office of Maisie A. Barringer</h1>
+        <nav>
+            <ul>
+                <li>Client Reviews</li>
+                <li>Schedule a meeting</li>
+                <li>Contact Me</li>
+                <li>Informational Videos</li>
+                <li>Services</li>
+            </ul>
+        </nav>
+    </header>
+    <div id="reviews"></div>
+    <div id="schedule"></div>
+    <div id="contact"></div>
+    <div id="videos"></div>
+    <div id="services">
+        <ul>
+            <li>Divorce</li>
+            <li>Custody</li>
+            <li>Mitigation</li>
+            <li>Adoption</li>
+            <li>Will Writing</li>
+        </ul>
+    </div>
```

Comment:
I think this looks great. We just need to start brainstorming and discussing the specific information we'd like to add. Also, you could add a footer element at the end for some extra _pizzazz_.

(btw I hope I'm doing this code review correctly)